Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG	GIT_VERSION=unknown
+ARG	BUILD_TARGET
+ARG	FROM_LOCAL_PARENT
+
+FROM	golang:1-bullseye	as	gobuild
+
+ARG TARGETARCH
+
+# Download c2patool needed to sign our C2PA manifest
+# We download it from any of our previous builds, because building c2patool from source is very slow with QEMU
+RUN	apt update && apt install -yqq \
+	curl \
+	ca-certificates \
+	&& curl https://build.livepeer.live/c2patool/0.6.2/c2patool-linux-${TARGETARCH}.tar.gz -o /c2patool.tar.gz \
+	&& tar xzf /c2patool.tar.gz
+
+WORKDIR	/src
+
+ADD	go.mod go.sum	./
+RUN	go mod download
+
+ADD	.	.
+RUN make build
+
+ARG	GIT_VERSION
+ENV	GIT_VERSION="${GIT_VERSION}"
+
+FROM	ubuntu:22.04	AS	catalyst
+
+ENV	DEBIAN_FRONTEND=noninteractive
+
+LABEL	maintainer="Amritanshu Varshney <amritanshu+github@livepeer.org>"
+
+ARG	BUILD_TARGET
+
+RUN	apt update && apt install -yqq wget software-properties-common \
+	&& add-apt-repository -y ppa:ubuntuhandbook1/ffmpeg6
+
+RUN	apt update && apt install -yqq \
+	curl \
+	ca-certificates \
+	musl \
+	python3 \
```

Comment:
Not sure we need all these dependencies. I guess it's for VOD. @thomshutt @mjh1 could you confirm we need those for vod?

Reply:
I'm not sure - other than ca-certificates we might be able to get rid of some of those others. It'll probably need a bit of trial and error though unfortunately

Reply:
Ok, I removed most of them (including ffmpeg), because I think we'll run VOD nodes still as the whole Catalyst bundle. And for the livestream API, we don't need it.